internal/exec/resolvable/resolvable.go

```diff
@@ -363,15 +367,42 @@ func (b *execBuilder) makeFieldExec(typeName string, f *types.FieldDefinition, m
 		}
 	}
 
+	directivesPackers := make([]*packer.StructPacker, 0, len(f.Directives))
+	for _, d := range f.Directives {
+		n := d.Name.Name
+
+		// FIXME: doesn't have a visitor, so can't be packed
+		// This could cause issues when present with other directives
+		// because we assume every directive gets a packer
+		if n == "deprecated" {
```

Comment:
Indeed `@skip`, `@include`, `@deprecated` and `@specifiedBy` are the only built-in directives. However, only the `@deprecated` directive is defined on `FIELD_DEFINITION`. I do not expect many more to be added in future. That is why, I think that it's fine to ignore it like this and and treat it in a special way.